### PR TITLE
split analytics dimension for browse templates

### PR DESCRIPTION
**Pull request by patrickpatrickpatrick**
## What

This PR is for splitting the navigation_page_type dimension into navigation_page_type and navigation_list_type.

## Why 
These changes were requested to make analytics better.

## Visual Changes
There should be none.

### CODE REVIEW

**Comment by patrickpatrickpatrick** [comment]
https://github.com/alphagov/govuk_publishing_components/pull/2722 needs to be merged also for changes to work

**Review by hannalaakso — APPROVED** [review]
I've tested this together with https://github.com/alphagov/govuk_publishing_components/pull/2722 and the correct dimensions are being set 🎉

File: app/views/browse/new_show.html.erb [hunk 1]
```diff
@@ -6,7 +6,8 @@
   <%= render "govuk_publishing_components/components/meta_tags", {
     content_item: {
       # TODO align this naming "level 1" vs "second_level" mismatch
```
Comment (hannalaakso) [inline app/views/browse/new_show.html.erb:8]: Nit: I'm pretty sure we can remove this comment, I think the fix here is that this refers to.

Comment (patrickpatrickpatrick) [reply app/views/browse/new_show.html.erb:8]: ah yes good spot

**Pull request merged**